src/ngx_str.hpp

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+extern "C" {
+#include <ngx_config.h>
+#include <ngx_core.h>
+}
+
+#include "str_view.hpp"
+
+StrView toStrView(ngx_str_t str)
+{
+    return StrView((char*)str.data, str.len);
+}
+
+template<class StrView>
```

Comment:
Let's use usual `T` and have a space after `template`. Also, it makes sense to start accepting `const T&` to avoid copying of types like `std::string`.

Reply:
See [ccb30e1](https://github.com/nginxinc/nginx-otel/pull/34/commits/ccb30e11659828c2fb9efa0229cecee0cdcb4237).